TheForceEngine/TFE_A11y/accessibility.cpp

```diff
@@ -178,7 +198,7 @@ namespace TFE_A11Y  // a11y is industry slang for accessibility
 		Caption caption;
 		caption.text = args[1];
 		caption.env = CC_Cutscene;
-		caption.msRemaining = caption.text.length() * 90 + 750;
+		caption.microsecondsRemaining = caption.text.length() * 90 + 750;
```

Comment:
Does this calculation need to be updated since you are changing from Milliseconds to Microseconds?

Reply:
@luciusDXL D'oh! Yes, I missed that function; it's only used when displaying a test caption from the console. I can fix it later today.